src/xdocs/report_issue.xml

```diff
@@ -174,8 +209,42 @@ expected level should be 12. [Indentation]
         The more examples the better.
       </p>
       <p>
-        If you have open source project, please send a link to the whole java file.
+        Explain the rationale behind implementing the new check with relevant
+        references (if possible).
       </p>
+      <p>
+        Feel free to propose a name for the new Check, along with all its attributes.
+      </p>
+      <p>
+        If you have open source project, please give a link to the whole java file.
+      </p>
+      <p>
+        Example of Check/Module Request that we expect:
+      </p>
+      <source>
+Describe the problem in detail:
+
+/var/tmp $ cat config.xml
+<module name="YourNewModule">
+  <property name="yourPossibleProperty1" value="true"/>
```

Comment:
it is not visible in HTML
![Screen Shot 2020-06-16 at 12 39 45 PM](https://user-images.githubusercontent.com/812984/84820448-83f1b100-afce-11ea-982e-dea70e2f73ac.png)

Reply:
fixed